<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGYRequestTool</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayout1">
    <item>
     <widget class="QLabel" name="labelRequestMethod">
      <property name="text">
       <string>请求方法</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="select_comboxBox">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>请选择</string>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="url_lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="send_pushButton">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear_pushButtion">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayout2">
    <item>
     <widget class="QLabel" name="labelRequestHead">
      <property name="text">
       <string>请求头</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelRequestBody">
      <property name="text">
       <string>请求体</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>701</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayout3">
    <item>
     <widget class="QPlainTextEdit" name="header"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="body"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayout4">
    <item>
     <widget class="QLabel" name="labelResponseHead">
      <property name="text">
       <string>响应头</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelReponseBody">
      <property name="text">
       <string>响应体</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>701</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayout5">
    <item>
     <widget class="QTextBrowser" name="resp_h_textBrowser"/>
    </item>
    <item>
     <widget class="QTextBrowser" name="resp_b_textBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
